spec/spec.md

```diff
@@ -388,7 +390,9 @@ where:
   - `uri`: (optional) a URI representing the web address associated with the dapp endpoint making this authorization request. If present, it must be an absolute, hierarchical URI.
   - `icon`: (optional) a relative path (from `uri`) to an image asset file of an icon identifying the dapp endpoint making this authorization request
   - `name`: (optional) the display name for this dapp endpoint
-- `chain`: (optional) if set, the [chain identifier](#chain-identifiers) for the chain with which the dapp endpoint intends to interact; supported values include `solana:mainnet`, `solana:testnet`, `solana:devnet`, `mainnet-beta`, `testnet`, `devnet`. If not set, defaults to `solana:mainnet`.
+- `chains`: (optional) if set, a list of the [chain identifiers](#chain-identifiers) for the chain with which the dapp endpoint intends to interact; supported values include `solana:mainnet`, `solana:testnet`, `solana:devnet`, `mainnet-beta`, `testnet`, `devnet`. If not set, defaults to `[solana:mainnet]`.
```

Comment:
hmm good question. This is where this change starts to become breaking. Adding a chain parameter to requests would be the safest way forward. it could be made optional, with some default assumed like the first chain in the list. But I'm not liking that. 

Historically (based on my readings from previous discussions), we've maintained a stance that each authorization scope should be for 1 chain, due to risk of reply attacks. I am thinking now that we should maintain this stance. 

If a dapp does wish to interact with multiple chains (for a bridging service for ex), it would instead call `authorize` for each chain. I don't see a problem with this and think this is a reasonable place for MWA to deviate a bit from ws. 

@jordansexton thoughts?